### PR TITLE
Expose ore multiplier to CraftTweaker

### PR DESCRIPTION
**Pull request by hjae78**
**What:**

This PR exposes the ore multiplier of a material to CraftTweaker.

### CODE REVIEW

**Review by htmlcsjs — APPROVED** [review]
going to wait for someone else to look over this before merge

**Review by DStrand1 — APPROVED** [review]

**Pull request merged**
